src/pytest/test.py

```diff
@@ -1965,80 +1868,13 @@ def assertResultsEqual(env, exp, got, inorder=True):
         exp_fields = to_dict(exp_fields)
         env.assertEqual(exp_fields, got_fields, message="at position {}".format(x))
 
-
-def testAlterIndex(env):
-    env.cmd('FT.CREATE', 'idx', 'SCHEMA', 'f1', 'TEXT')
-    env.cmd('FT.ADD', 'idx', 'doc1', 1.0, 'FIELDS', 'f1', 'hello', 'f2', 'world')
-    env.cmd('FT.ALTER', 'idx', 'SCHEMA', 'ADD', 'f2', 'TEXT')
-    env.cmd('FT.ADD', 'idx', 'doc2', 1.0, 'FIELDS', 'f1', 'hello', 'f2', 'world')
-    for _ in env.retry_with_reload():
-        ret = env.cmd('FT.SEARCH', 'idx', 'world')
-        env.assertEqual([1, 'doc2', ['f1', 'hello', 'f2', 'world']], ret)
-
-    env.cmd('FT.ALTER', 'idx', 'SCHEMA', 'ADD', 'f3', 'TEXT', 'SORTABLE')
-    for x in range(10):
-        env.cmd('FT.ADD', 'idx', 'doc{}'.format(x + 3), 1.0,
-                 'FIELDS', 'f1', 'hello', 'f3', 'val{}'.format(x))
-
-    for _ in env.retry_with_reload():
-        # Test that sortable works
-        res = env.cmd('FT.SEARCH', 'idx', 'hello', 'SORTBY', 'f3', 'DESC')
-        exp = [12, 'doc12', ['f1', 'hello', 'f3', 'val9'], 'doc11', ['f1', 'hello', 'f3', 'val8'], 'doc10', ['f1', 'hello', 'f3', 'val7'], 'doc9', ['f1', 'hello', 'f3', 'val6'], 'doc8', ['f1', 'hello', 'f3', 'val5'], 'doc7', [
-                'f1', 'hello', 'f3', 'val4'], 'doc6', ['f1', 'hello', 'f3', 'val3'], 'doc5', ['f1', 'hello', 'f3', 'val2'], 'doc4', ['f1', 'hello', 'f3', 'val1'], 'doc3', ['f1', 'hello', 'f3', 'val0']]
-
-        assertResultsEqual(env, exp, res)
-
-    # Test that we can add a numeric field
-    env.cmd('FT.ALTER', 'idx', 'SCHEMA', 'ADD', 'n1', 'NUMERIC')
-    env.cmd('FT.ADD', 'idx', 'docN1', 1.0, 'FIELDS', 'n1', 50)
-    env.cmd('FT.ADD', 'idx', 'docN2', 1.0, 'FIELDS', 'n1', 250)
-    for _ in env.retry_with_reload():
-        res = env.cmd('FT.SEARCH', 'idx', '@n1:[0 100]')
-        env.assertEqual([1, 'docN1', ['n1', '50']], res)
-
-    env.expect('FT.ALTER', 'idx', 'SCHEMA', 'NOT_ADD', 'f2', 'TEXT').error()
-    env.expect('FT.ALTER', 'idx', 'SCHEMA', 'ADD').error()
-    env.expect('FT.ALTER', 'idx', 'SCHEMA', 'ADD', 'f2').error()
-
-def testAlterValidation(env):
```

Comment:
?

Reply:
remove support

Reply:
We decided it will not be removed right?